src/components/screens/home/HomeComponent.js

```diff
@@ -0,0 +1,70 @@
+// @flow
+
+import React, { Fragment, Component } from 'react';
+import { View } from 'react-native';
+
+import CONSTANTS from '../../../utils/CONSTANTS';
```

Comment:
Didn't we add some library to allow this? Just to check :)
```suggestion
import CONSTANTS from '~/utils/CONSTANTS';
```